release-notes.html.haml

```diff
@@ -2,6 +2,69 @@
 layout: project
 title: Release Notes
 ---
+%a{:name => "9.2"}
+.jumbotron
+  %h2
+    Infinispan 9.2 Release Notes
+    %span{:style => "font-size: 0.4em"}
+
+.row
+  .col-md-6
+    %i.enormous-icon.fa-th-large.pull-left.icon-2x
+    %h3 Core
+    %ul
+      %li Further improvements to conflict resolution and merge policies
+      %li Reactive streams for increased throughput
+      %li Biased reads for scattered caches
+      %li More accurate off-heap memory calculations
+    %i.enormous-icon.icon-wrench.pull-left.icon-2x
+    %h3 API
+    %ul
+      %li Multimap caches for embedded and Hot Rod
+      %li Clustered Counters for Hot Rod
+      %li Clustered Locks
+      %li Wildcard configurations
+      %li Cluster-wide cache admin via embedded and remote
```

Comment:
* Added CacheStream.collect(Supplier) for more user-friendly clustered collect functionality